weave-js/src/components/PagePanelComponents/Home/Browse3/pages/wfReactInterface/utilities.ts

```diff
@@ -287,6 +287,15 @@ export const objectVersionNiceString = (ov: ObjectVersionSchema) => {
   return result;
 };
 
+export const isObjDeleteError = (e: any): boolean => {
```

Comment:
seems like we can do better here?

Reply:
can we pass status code / other info that does not rely on regexing the string?

Reply:
Ugh can we make up a status code for "you deleted this"? I totally agree it does seem like we can do better here, except really we want to 404 on the backend which doesn't tell us much. The current `trace_server.py` error handlers have no prior art for passing extra stuff, but we could sneak some extra info there? 

maybe:
```python
@app.exception_handler(ObjectDeletedError)
def object_deleted_error_handler(request: Request, exc: ObjectDeletedError):
    exc_str = str(exc)
    try:
        json.loads(exc_str)
    except json.JSONDecodeError:
        exc_str = {"reason": exc_str, "deleted_at": exc.deleted_at}
    return JSONResponse(
        status_code=404,
        content=exc_str,
    )
```